### PR TITLE
Added sticky toolbar documentation

### PR DESCRIPTION
**Pull request by wachterjohannes**
Main PR: https://github.com/sulu-io/sulu/pull/1821

### CODE REVIEW

File: bundles/admin/javascript-hooks/sticky-toolbar.rst [hunk 1]
```diff
@@ -0,0 +1,24 @@
+Sticky-Toolbar
+==============
+
+The sticky-toolbar extension enables the datagrid list toolbar to sick under the
```
Comment (danrot) [inline bundles/admin/javascript-hooks/sticky-toolbar.rst]: `datagrid list toolbar to stick`

File: bundles/admin/javascript-hooks/sticky-toolbar.rst [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+header the page will be scrolled.
```
Comment (danrot) [inline bundles/admin/javascript-hooks/sticky-toolbar.rst]: `header, when the page will be scrolled.`

File: bundles/admin/javascript-hooks/sticky-toolbar.rst [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        initialize: function() {
+            this.sandbox.stickyHeader.enable(this.$el);
+        },
+
+        destroy: function() {
+            this.sandbox.stickyToolbar.disable(this.$el);
+        }
+    }
+
+Optional you can add a custom scroll edge to customize the behaviour. To reset
```
Comment (danrot) [inline bundles/admin/javascript-hooks/sticky-toolbar.rst]: What's a custom scroll edge? How is that done?

File: bundles/admin/javascript-hooks/sticky-toolbar.rst [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+.. code-block:: javascript
+
+    {
+        initialize: function() {
+            this.sandbox.stickyHeader.enable(this.$el);
+        },
+
+        destroy: function() {
+            this.sandbox.stickyToolbar.disable(this.$el);
+        }
```
Comment (danrot) [inline bundles/admin/javascript-hooks/sticky-toolbar.rst]: Wouldn't it be nice if that would be handled by Sulu somehow? Would be nice if the component just has something like this:

```javascript
{
    stickyHeader: true
}
```

File: bundles/admin/javascript-hooks/sticky-toolbar.rst (hunk 3 above)
Comment (wachterjohannes) [inline bundles/admin/javascript-hooks/sticky-toolbar.rst]: if you have tabs you have to scroll 50px more to snap in the toolbar.

**Comment by wachterjohannes** [comment]
@danrot i have fixed your comments

File: bundles/admin/javascript-hooks/sticky-toolbar.rst [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+header and 50 for the tabs-container).
+
+.. code-block:: javascript
+
+    {
+        stickyToolbar: true // or a number like: 140
+    }
+
+To reset the scroll-container if you rerender your component you can call the
+`reset` method.
```
Comment (danrot) [inline bundles/admin/javascript-hooks/sticky-toolbar.rst]: `this.sandbox.stickyFooter.reset`, otherwise you don't know how to use this method.

Comment (wachterjohannes) [inline bundles/admin/javascript-hooks/sticky-toolbar.rst]: fixed

**Pull request merged**